### PR TITLE
Add README blurb to the two decoders

### PR DESCRIPTION
**Pull request by Jawnnypoo**
Similar to [Retrofit adapters](https://github.com/square/retrofit/tree/master/retrofit-converters/moshi), I think having a README in each of the "extension" modules would be useful for those who potentially look at the project on GitHub, but don't look at the docs site.

### CODE REVIEW

**Comment by Jawnnypoo** [comment]
I added to this to add the SVGs docs to the site as well.

File: docs/svgs.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Gifs
```
Comment (colinrtwhite) [inline docs/svgs.md]: Should be: SVGs

**Comment by colinrtwhite** [comment]
Good idea - left one comment.

File: coil-gif/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# coil-gif
```
Comment (colinrtwhite) [inline coil-gif/README.md]: Let's move the existing `docs/gifs.md` to `coil-gif/README.md` and we can copy it back during the deploy process.

File: coil-svg/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+implementation("io.coil-kt:coil-svg:latest.version.here")
+```
+Note that this imports [AndroidSVG](https://bigbadaboom.github.io/androidsvg/) as well.
+
+## Quick Start
+
+Add the decoders to your component registry when constructing your ImageLoader:
+
+```kotlin
+// within a global place, like your Application.onCreate()
```
Comment (colinrtwhite) [inline coil-svg/README.md]: Nit: https://nedbatchelder.com/blog/201401/comments_should_be_sentences.html

**Comment by colinrtwhite** [comment]
Can you update https://github.com/coil-kt/coil/blob/master/deploy_docs.sh to copy the readme files to the docs folder during the deploy process? Thanks!

File: coil-gif/README.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+Adds GIF decoding support to Coil.
+
+```kotlin
+implementation("io.coil-kt:coil-gif:latest.version.here")
```
Comment (colinrtwhite) [inline coil-gif/README.md]: Let's hardcode the latest version. We can find/replace during the deploy process.

File: coil-svg/README.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
+# SVGs
+
+To add SVG support, import the extension library:
+
+```kotlin
+implementation("io.coil-kt:coil-svg:0.6.1")
+```
+
+And add the decoders to your component registry when constructing your `ImageLoader`:
```
Comment (colinrtwhite) [inline coil-svg/README.md]: *decoder

**Pull request merged**